quickstart/101-attestation-provider/providers.tf

```diff
@@ -0,0 +1,15 @@
+terraform {
+
+  required_version = ">=0.12"
+  
+  required_providers {
+    azurerm = {
+      source = "hashicorp/azurerm"
```

Comment:
Can we run `terraform fmt` command for this directory please? The formatted line should be:

```hcl
source  = "hashicorp/azurerm"
```